### ISSUE TITLE
Cut and copy operations not working or inconsistent in the Custom HTML and Code blocks (and possibly other ones?)

### ISSUE THREAD

**Issue by afercia** [WordPress/gutenberg#4269]
Basic operations like copying, cutting, and pasting content should work flawlessly and in a consistent way across blocks, always giving the expected result. Seems this is not happening for some blocks. 

To reproduce:
- add a Custom HTML block
- enter some text
- press Cmd (or Ctrl) + A to select the block content
- press Cmd (or Ctrl) + X to cut the content: nothing happens (not even with a partial selection)

The content stays in the block, instead it should disappear. It gets actually copied: paste in an external editor and you'll see it was copied to the clipboard.

I'm getting the same result in the Code block.

Additionally, the copied text contains also the Gutenberg comments, and it should not. Copying from other blocks doesn't do that. For example, here's the content I get copying from different kind of blocks:

Custom HTML (includes the comments):
```
<!-- wp:html -->
Hello
<!-- /wp:html -->
```

Code (contains also the pre and code tags):
```
<!-- wp:code -->
<pre class="wp-block-code"><code>Hello</code></pre>
<!-- /wp:code -->
```

Paragraph:
```
Hello
```

List:
```
Hello
Hello
Hello
```

**Comment by iseulde** [comment]
@afercia Does #4271 fix this for you? It seems to do so here.

**Comment by afercia** [comment]
@iseulde seems it fixes it for me 🎉

**Comment by iseulde** [comment]
Awesome! Closing as I just merged #4271.
